geonode/maps/urls.py

```diff
@@ -49,6 +49,7 @@
     url(r'^new/data$', 'new_map_json', name='new_map_json'),
     url(r'^checkurl/?$', 'ajax_url_lookup'),
     url(r'^snapshot/create/?$', 'snapshot_create'),
+    url(r'^new/copy/(?P<mapid>[^/]+)$', existing_map_view, name='map_view'),
```

Comment:
There is already map_view, we should use unique name. Set it to `copy_map` like my another comment